### DISCUSSION TITLE
MapLibre Newsletter

### DISCUSSION THREAD

**Discussion by wipfli** [maplibre/flutter-maplibre-gl#350] [Ideas]
Hey Flutters

I am writing a monthly newsletter for the MapLibre Org and it is always so cool to see how much active flutter-maplibre-gl has. 

Would you be interested in writing a paragraph about recent developments in your repo for the December newsletter?

It could be something small just like here: https://maplibre.org/news/2023-11-30-maplibre-newsletter-november-2023/#martin

**Comment by m0nac0** [comment]
Sure, great idea!

Some things coming to my mind (feel free to rewrite):

We published two new releases recently: [0.18.0](https://github.com/maplibre/flutter-maplibre-gl/releases/tag/0.18.0) and [0.17.0](https://github.com/maplibre/flutter-maplibre-gl/releases/tag/0.17.0).

0.18.0 contains a major new feature: you can now set arbitrary layer properties at runtime (https://github.com/maplibre/flutter-maplibre-gl/pull/303). We also improved the documentation for users and contributors (updated old docs and added new docs) and work continues on pushing releases to pub.dev (The version currently published on pub.dev does not work on the web https://github.com/maplibre/flutter-maplibre-gl/issues/309).

0.17.0 contains a lot of new features and bugfixes (too many to list all of them here) that lived in the repo for some time but had not been "officially released" so far. 0.17.0 also contains some breaking changes, but the release notes contain instructions on how to easily fix them. Also, developers do not need to adapt their Podfile for iOS apps anymore as it was previously described in the Readme. Developers who previously added these lines should remove them, since not removing these lines may cause a build failure on iOS.
